### DISCUSSION TITLE
Do meeting notes fall under the scope of agile?

### DISCUSSION THREAD

**Discussion by natebass** [InnovateForVegas/doc-agile-for-volunteers#1] [Q&A]
Is agile for volunteers the right place to discuss the structure and guidelines for taking meeting notes?

I was inspired by notes from a project called [web-platform-tests](https://github.com/web-platform-tests/wpt-notes/blob/master/minutes/2024-08-06.md) and it got me thinking of ways we could improve our process at Open Sacramento. I can make a user story based on this.

@DanHugoDanHugo Let me know what you think.

**Comment by DanHugoDanHugo** [comment]
Agile, and XP before, emphasize communication and collaboration as fundamentals (Whole Team, brief stand-up meetings, the 3C´s of User Stories including the _Conversation_, etc).

_Agile for Volunteers_ does inherit these fundamental notions, but also assumes that volunteers may or may not show up for any reason, and in fact they might disappear completely. Meetings are thus less formal as attendance is never going to be reliable, outcomes of any meeting should be captured in the initiative planning and component project task scheme for action, with discussion (as issue comments, and/or actual Discussion using GitHub tools for example) closer to the particular item rather than in notes for meetings.

As well, the small-teams notion and the completely asynchronous nature of volunteer schedules makes a formal meeting structure difficult (this may vary per initiative or component project, it really depends on the individuals that make up that team), and meeting minutes are usually for people who missed the meeting (and for posterity). We would spend 100% of time keeping minutes for meetings not-attended…

Taking a quick glance through the web-platform-tests meeting minutes per your link, I have to say that I would be astonished if that level of detail (which includes tremendous contextual knowledge implied) would be consumed by a casual potential contributor… if we look at one example selected at random, [2023-11-07.md](https://github.com/web-platform-tests/wpt-notes/blob/master/minutes/2023-11-07.md) includes named github users with tasks and comments and whatnot, left as an exercise the the reader to pursue. These are most definitely not episodic minutes/notes, nor should they be required to be, but they offer less useful information (or more work) to someone who lacks the context of the overall story arc of the project(s), meetings, interactions, issues opened and closed, etc etc etc.

One additional in-context note, placing meeting minutes into date-named markdown files is as enticing as placing project documentation into a doc/locale directory structure… navigating the data to find the information is already allegedly difficult with a table of contents and named files. There is probably a need for some tools, with AI all the rage now, to roll up meeting minutes, huge slack channels, and similar into usable conclusions. The conversation that leads to a conclusion is valuable, but is spread out by date, attendance, and topics of discussion in a meeting minutes approach.

From my own experience working with volunteers, reading is not very popular, so the focus is on linking plans and tasks takes precedence over meeting minutes (Iteration retrospectives seem like they might be more useful, and there is a Discussion template for this), and the *Atomic Teams* notion is built around the presumption that volunteers will disappear, with *Persistent Teams* more likely to benefit from formalized or structured meetings with minutes and notes, though anyone is welcome to capture and publish meeting minutes for any meeting so long as they are in keeping with the Code of Conduct.

A new document for Meetings could be added to the Agile for Volunteers documentation to expound on the pros and cons of meetings, minutes, summaries, outcomes, issues and discussions (or platform-specific tool use), etc. These would be suggestions rather than rules, of course.

@natebass
